src/Compilers/Core/Portable/SourceGeneration/GeneratorSyntaxWalker.cs

```diff
@@ -33,11 +32,14 @@ public void VisitWithModel(Lazy<SemanticModel>? model, SyntaxNode node)
             _semanticModel = null;
         }
 
-        public override void Visit(SyntaxNode node)
+        public void Visit(SyntaxNode node)
```

Comment:
>public [](http://example.com/codeflow?start=8&length=6)

`private`? It looks like the instance won't be properly configured, unless this method is called from ```VisitWithModel```.

Reply:
There's no real need to have the separate Visit method, I had just kept it because that's the way it was before. Merging the two in commit 5.